css/components/navigation.css

```diff
@@ -0,0 +1,90 @@
+.nav-bar {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: var(--primary-color);
+  padding: 1rem;
+}
+.nav-section-items i {
+  font-size: 1.5rem;
+  color: var(--white-color);
+}
+.nav-section {
+  display: flex;
+  align-items: center;
+}
+.nav-section-items a {
+  color: var(--white-color);
+}
+.nav-logo h1 {
+  color: var(--white-color);
+}
+.search-bar {
+  background-color: var(--white-color);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-radius: 4px;
+}
+.searchbar-input {
+  border: none;
+  background-color: transparent;
+  padding: 0.8rem 2rem;
+  outline: none;
```

Comment:
Instead of outline:none you can use
outline: 2px solid transparent;
outline-offset: 2px;